myuw/static/css/mobile.less

```diff
@@ -1399,6 +1416,13 @@ label { font-weight: normal; margin-bottom: auto; }
 				    }
 			    }
 		    }
+		    .card-badge-container .no-top-margin {
+				padding-top: 0;
+	    	}	
+		    .card-badge-container.tuition {
```

Comment:
Let's investigate whether this pseudoselector syntax is correct. I unfortunately am not certain but this doesn't look right to me.